### PR TITLE
 WORKS AND ABOUT MYSELF SECTION

### PR DESCRIPTION
**Pull request by Yorsyboy**
## Description
In this PR, I implemented the works and about myself section
### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.

Thank you for taking the time to review this pull request. :pray:

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -114,3 +115,328 @@ ul.social {
...
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 88px 0;
+}
+
```
Comment (AmaduKamara) [inline main.css:138]: [ Required Changes ]
You're doing great bro, 👍.

- [ ] Please add margin-left and margin-right for each project so your design will match the Figma design.
Your design
image

The Figma Design
image

Please fix that.
Hint: you can replace your line `136` with `margin: 88px 20px;`  🤝

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️
Hi @Yorsyboy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF


## Highlights
✔️ No Linter error
✔️ Professional README
✔️ Good descriptive PR 
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css [hunk 2]
```diff
@@ -114,3 +115,328 @@ ul.social {
...
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 88px 20px;
```
Comment (BenMukebo) [inline main.css:136]: - [ Optional] I think you should set the `margin: 88px auto;`, this will make your cards be adjusted equally on the vertical axis.

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### *** Hi @Yorsyboy *** , 

### **Congratulations! 🎉 Project Approved ✅**



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css (hunk 2 above)
Comment (Yorsyboy) [reply main.css:136]: Thanks I will do that in my next commit

**Pull request merged**